accounts/abi/abi_test.go

```diff
@@ -738,12 +738,12 @@ func TestBareEvents(t *testing.T) {
 // TestUnpackEvent is based on this contract:
 //
 //	contract T {
-//	  event received(address sender, uint amount, bytes memo);
-//	  event receivedAddr(address sender);
-//	  function receive(bytes memo) external payable {
-//	    received(msg.sender, msg.value, memo);
-//	    receivedAddr(msg.sender);
-//	  }
+//		event received(address sender, uint amount, bytes memo);
+//		event receivedAddr(address sender);
+//		function receive(bytes memo) external payable {
+//			received(msg.sender, msg.value, memo);
+//			receivedAddr(msg.sender);
+//		}
 //	}
```

Comment:
nit: I'm seeing some more spacing fixes on this file

Reply:
I tried to minimize the diff between coreth:
https://github.com/ava-labs/coreth/blob/master/accounts/abi/abi_test.go